helpers/auth.py

```diff
@@ -200,6 +200,17 @@ def login(self):
         if self.config.auth_username:
             username_element = self.fill_username()
 
+        if self.config.auth_form_type == 'MULTI_STEP':
+            log(f"Waiting for {self.config.auth_password_field_name} element to load")
+            sleep(5)
```

Comment:
sleep in here and on line 212 so it has time to give time to render the next element (password)

Reply:
pull that value to a constant - or configurable parameter would be even better